interfaces/builtin/block_devices.go

```diff
@@ -60,6 +60,7 @@ const blockDevicesConnectedPlugAppArmor = `
 /dev/i2o/hdd[a-x] rw,                    # I2O hard disk continued
 /dev/mmcblk[0-9]{,[0-9],[0-9][0-9]} rw,  # MMC (up to 1000 devices)
 /dev/nvme[0-9]{,[0-9]} rw,               # NVMe (up to 100 devices)
+/dev/nvme[0-9]{,[0-9]}n[0-9]{,[0-9]} rw, # NVMe (up to 100 devices)
```

Comment:
kubernetes_support.go uses this pattern:
```
# nvme0-99, 1-63 partitions with 1-63 optional namespaces
mount /dev/nvme{[0-9],[1-9][0-9]}n{[1-9],[1-5][0-9],6[0-3]}{,p{[1-9],[1-5][0-9],6[0-3]}} -> /var/snap
```

And one in raw_volume.go (though that's a Go regexp):
```
const nvmePat = `nvme([0-9]|[1-9][0-9])(n([1-9]|[1-5][0-9]|6[0-3])){0,1}p([1-9]|[1-5][0-9]|6[0-3])`
```

Reply:
good catch I think I'll use the one from k8s-support, though we don't need the partitions here